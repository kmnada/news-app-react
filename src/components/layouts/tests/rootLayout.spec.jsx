import React from 'react';
import { shallow } from 'enzyme';
import RootLayout from '../rootLayout';

describe(' RootLayout component', () => {
  it('renders default component', () => {
    const wrapper = shallow(<RootLayout />);
    expect(wrapper).toMatchSnapshot();
  });
});
