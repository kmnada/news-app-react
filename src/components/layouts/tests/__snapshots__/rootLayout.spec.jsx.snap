// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` RootLayout component renders default component 1`] = `ShallowWrapper {}`;
